planetary-social/nos
# Remove stories to decrease the number of downloaded events

## Issues covered
#1309

## Description
This removes the stories UI entirely in order to decrease the number of downloaded events. I've seen a 3-4x reduction in events, as measured by events left in the parse queue.

## How to test
1. Navigate to Feed
2. Observe

After navigating to Feed, also look at the events in the console, specifically around how many are left in the parse queue. Compared to `main`, there should be a huge reduction.

## Screenshots/Video
| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 Pro - 2024-07-29 at 17 44 47](https://github.com/user-attachments/assets/1345a425-2679-4c44-8a56-ecf6526759cf) | ![Simulator Screenshot - iPhone 15 Pro - 2024-07-29 at 17 39 36](https://github.com/user-attachments/assets/d8b39fe0-f566-4e1a-9292-23a6d387022d) |
